### PR TITLE
[File Manager] Fixed viewing files

### LINKED ISSUE #61: FileManager can't open selected files by intent action
I've been following latest documentation and tried to open the files after all permissions check mentioned in the docs. I thought I might be missing something, ergo, after checking the FileManager sample, it seems `openFile()` extension from FileManager sample is also helpless in opening the file. I tried to open a word file (.docx) an image file, a pdf file. It opens the app but the file can't be found and (Adobe Reader, Word and Gallery shows the main screen).

Tested with Realme 5 Pro (Android 10) with all permissions granted

```
fun openFile(activity: AppCompatActivity, selectedItem: File) {
    // Get URI and MIME type of file
    val uri = Uri.fromFile(selectedItem).normalizeScheme()
    val mime: String = getMimeType(uri.toString())

    // Open file with user selected app
    val intent = Intent()
    intent.action = Intent.ACTION_VIEW
    intent.data = uri
    intent.type = mime
    return activity.startActivity(intent)
}
```

### PR DESCRIPTION
**Pull request by mirwisek**
Resolves #61

### CODE REVIEW

File: FileManager/app/src/main/java/com/android/samples/filemanager/FileUtils.kt [hunk 1]
```diff
@@ -19,10 +19,14 @@ package com.android.samples.filemanager
 import android.content.Intent
 import android.net.Uri
```
Comment (cketti) [inline FileManager/app/src/main/java/com/android/samples/filemanager/FileUtils.kt:20]: Unused import

File: FileManager/app/src/main/java/com/android/samples/filemanager/FileUtils.kt [hunk 2]
```diff
@@ -19,10 +19,14 @@ package com.android.samples.filemanager
...
 import android.os.Environment
+import android.util.Log
```
Comment (cketti) [inline FileManager/app/src/main/java/com/android/samples/filemanager/FileUtils.kt:22]: Unused import

**Review by cketti — APPROVED** [review]
Nice :+1:

To make it easier on the maintainers you should give this pull request a meaningful title, e.g. `[File Manager] Fix viewing files`.

It's also helpful when the pull request description contains a few sentences on what the change actually does.

Suggestion:
> Use `FileProvider` to obtain a `content://` URI that can be used with the `ACTION_VIEW` Intent. This is the recommended approach and much more likely to work than using a `file://` URI.

**Review by mirwisek — COMMENTED** [review]
- Pull request has been assigned name
- Other issues have been resolve (Removed unused imports)

File: FileManager/app/src/main/java/com/android/samples/filemanager/FileUtils.kt (hunk 1 above)
Comment (nic0lette) [reply FileManager/app/src/main/java/com/android/samples/filemanager/FileUtils.kt:20]: This one is still unused, so reopening.

File: FileManager/app/src/main/java/com/android/samples/filemanager/FileUtils.kt [hunk 3]
```diff
@@ -19,9 +19,12 @@ package com.android.samples.filemanager
...
 import android.webkit.MimeTypeMap
 import androidx.appcompat.app.AppCompatActivity
-import java.io.File
```
Comment (nic0lette) [inline FileManager/app/src/main/java/com/android/samples/filemanager/FileUtils.kt:24]: This import is required below (see line 32).

File: FileManager/app/src/main/res/xml/provider_paths.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (nic0lette) [inline FileManager/app/src/main/res/xml/provider_paths.xml:1]: Please add the Apache 2.0 license header that's found in the other files. :)

**Review by nic0lette — CHANGES_REQUESTED** [review]
Thanks for the PR -- just a few minor things I'd appreciate if you fixed up.

Also, would you be able to rebase this onto the current main? Thank you!

File: FileManager/app/src/main/java/com/android/samples/filemanager/FileUtils.kt (hunk 3 above)
Comment (mirwisek) [reply FileManager/app/src/main/java/com/android/samples/filemanager/FileUtils.kt:24]: Fixed in 25399f45767d7144815617ba012eb04bc9c0b449

File: FileManager/app/src/main/res/xml/provider_paths.xml (hunk 4 above)
Comment (mirwisek) [reply FileManager/app/src/main/res/xml/provider_paths.xml:1]: Added here 683451ad760cdcbd17b158358a0ec27f267e4f07

**Review by mirwisek — COMMENTED** [review]
Changes have been made as requested

**Review by nic0lette — APPROVED** [review]

**Comment by nic0lette** [comment]
Thank you!

**Pull request merged**